interface/forms/eye_mag/report.php

```diff
@@ -33,7 +33,7 @@
 require_once(dirname(__FILE__) . "/../../../library/api.inc");
 require_once(dirname(__FILE__) . "/../../../library/lists.inc");
 require_once(dirname(__FILE__) . "/../../../library/forms.inc");
-require_once(dirname(__FILE__) . "/../../../library/patient.inc");
+require_once(dirname(__FILE__) . "/../../../library/patient.inc.php");
```

Comment:
is the .inc not working? (note there are quite a few .inc scripts included above here)